caravel/models.py

```diff
@@ -933,6 +933,10 @@ def get_datasources(self):
         return json.loads(requests.get(endpoint).text)
 
     def refresh_datasources(self):
+        endpoint = (
+            "http://{obj.coordinator_host}:{obj.coordinator_port}/status"
+        ).format(obj=self)
+        self.druid_version = json.loads(requests.get(endpoint).text)['version']
```

Comment:
I suspect you have to mock this call in the tests otherwise you'll get a failure